src/identities/mnemonic.ts

```diff
@@ -1,30 +1,177 @@
+import { confidential, ECPair, payments, Psbt } from 'liquidjs-lib';
 import * as bip39 from 'bip39';
+import { fromSeed as slip77fromSeed, Slip77Interface } from 'slip77';
+import { fromSeed as bip32fromSeed, BIP32Interface } from 'bip32';
 import Identity, {
   AddressInterface,
   IdentityInterface,
   IdentityOpts,
+  IdentityType,
 } from '../identity';
+import { Output } from 'liquidjs-lib/types/transaction';
+import { UnblindOutputResult } from 'liquidjs-lib/types/confidential';
+
+export interface MnemonicOptsValue {
+  mnemonic: string;
+  language?: string;
+  passphrase?: string;
```

Comment:
bip39 passprhase is kind of broken concept, in the sense is a nightmare for compatibility with other wallets. I would not take into account and assume it is always in clear text. Having the bip39 passphrase here, since we want it toghehter, there is no security benefit. Will leave to devlopers to decrypt and give us the mnemonic in plain text directly